lua/harpoon/buffer.lua

```diff
@@ -63,7 +63,7 @@ function M.setup_autocmds_and_keymaps(bufnr)
         bufnr,
         "n",
         "<ESC>",
-        "<Cmd>lua require('harpoon.buffer').run_toggle_command('<ESC>')<CR>",
+        "<Cmd>lua require('harpoon.buffer').run_toggle_command('Esc')<CR>",
```

Comment:
The angle brackets seem to get interpreted as a <CR> here, which causes an entry to be selected when closing the menu with `Esc` or `q`